ansible-st2/roles/mistral/meta/main.yml

```diff
@@ -1,25 +0,0 @@
----
-galaxy_info:
-  description: Install custom OpenStack Mistral, patched by StackStorm
-  author: armab
-  company: StackStorm
-  license: Apache
-  min_ansible_version: 1.9.1
-  platforms:
-    - name: Ubuntu
-      versions:
-        - trusty
-        - precise
-  categories:
-    - system
-dependencies:
-  - role: ANXS.postgresql
-    version: v1.2.1
-    tags: [db, postgresql]
-    sudo: yes
-    postgresql_databases:
-      - name: mistral
-    postgresql_users:
```

Comment:
We need to check the users and passwords setup in the db and change them in production.